--- test_registry.py.old	2024-07-08 01:55:31.224704990 -0700
+++ test_registry.py.new	2024-07-08 01:55:57.256042814 -0700
@@ -641,7 +641,7 @@
             return True, {}
 
         with io.open(get_data_path('real_file')) as fh:
-            if platform == "win32":
+            if platform == "xxxxx":
                 fh.seek(3)
                 self.registry.sniff(fh)
                 self.assertEqual(fh.tell(), 3)
